styles/index.css

```diff
@@ -46,9 +109,91 @@
 .left-corner,
 .readmore-button {
   font-family: var(--paragraph-sans);
-  font-weight: 400; /* Open Sans 400 for regular paragraphs */
 }
 .left-corner h1 {
-  font-family: var(--heading-sans);
-  font-weight: 600;
+  font-family: var(--paragraph-sans);
+}
+.left-corner p {
+  margin-left: 0;
+}
+.blogs-section-container {
+  padding: 30px 88px;
+  font-family: var(--paragraph-sans);
+}
+.blogs-header-container {
+  display: flex;
+  flex-direction: column;
+}
+.blogs-header-container h1,
+.blogs-header-container p {
+  margin: 10px;
+  text-align: center;
+}
+
+.blogs-carousel {
+  display: flex;
+  justify-content: flex-end;
+  gap: 10px;
+  margin-bottom: 10px;
+}
+
+.left-button,
+.right-button {
+  width: 15px;
+  height: 15px;
+  border: none;
+  background-color: inherit;
+  cursor: pointer;
+  padding: 0;
+  margin: 0;
+}
+
+.arrow-icon {
+  width: 100%;
+}
+
+.blog-post {
+  width: 20%;
+}
+.banner-image {
+  width: 100%;
+  object-fit: cover;
+  aspect-ratio: 3 / 2;
+}
+
+.blog-post-container {
+  display: flex;
+  gap: 20px;
+  justify-content: space-between;
+  overflow: hidden;
+}
+
+.blog-post-container a {
+  text-decoration: none;
+}
+
+.blog-post-container a:visited {
```

Comment:
why is it neccessary?

Reply:
This ensures the visited links aren't purple.